Update DeadPMTID mask for WCSim

S81D opened the pull request:
Looking at recent (since 2023) data, there are two PMTs (IDs **359** and **408**) that show no (or very little) activity. The way we simulate dead PMTs from WCSim is to simply mask out their hits in the `LoadWCSim` tool. To accurately model our data we therefore need to add them to the dead list.

The activity loss in these PMTs are observed in the AmBe source data statistics (across many runs):

Total hits: `2235737`
Average hit rate per PMT (assuming 121 PMTs equally participating): `18477 (0.83%)`

Some examples of active channels:

- Channel 332: `14532 (0.65%)`
- Channel 339: `13775 (0.62%)`
- Channel 404: `17433 (0.78%)`

The two PMTs in question:

- Channel **359**: `408 (0.018%)`
- Channel **408**: `0 (0%)` , i.e. dead as a doornail

We see a similar statistical behavior in the beam data. The mapping of the dead PMT list references [this file](https://github.com/ANNIEsoft/ToolAnalysis/blob/Application/configfiles/LoadWCSim/Chankey_WCSimID_v7.txt).

## Describe your changes

## Checklist before submitting your PR
- [X] This PR implements a single change (one new/modified Tool, or a set of changes to implement one new/modified feature)
- [X] This PR alters the minimum number of files to affect this change
- [N/A] If this PR includes a new Tool, a README and minimal demonstration ToolChain is provided
- [N/A] If a new Tool/ToolChain requires model or configuration files, their paths are not hard-coded, and means of generating those files is described in the readme, with examples provided on /pnfs/annie/persistent
- [N/A] For every `new` usage, there is a reason the data must be on the heap
- [N/A] For every `new` there is a `delete`, unless I explicitly know why (e.g. ROOT or a BoostStore takes ownership)

## Additional Material
More information on the dead PMT rates can be found [here](https://annie-docdb.fnal.gov/cgi-bin/sso/RetrieveFile?docid=5962&filename=MC%20Tuning%20and%20Validation%20_%20March%202025%20Collab%20Meeting%20_%20Steven%20Doran.pdf&version=1) (slide 33).

A good ToDo would be to determine when exactly these PMTs showed this behavior.

S81D committed:
Update DeadPMTIDs_p2v7.txt

Add two PMTs to the deadlist based on observed occurrence of hits in data

jminock reviewed (approved):
Excellent find and thank you for making this! Tagging this for a software/analysis meeting so others can hear about this. Otherwise, ready for merge.

S81D commented:
Here's an update following software meeting discussion:

There's **3** PMTs added to the dead list (359, 408, and 343) - i forgot to add rates for 343. The AmBe data shows < 0.01% occupancy for PMT 343 so it should be added to the list.

To double check all of these rates for high energy events (maybe they are very inefficient for single PE hits but fine for multi-PE), I have selected a sample of "muon-like" events from 3 different beam epochs to compare the rates. The muon like events are chosen with basic selection cuts:

- cluster charge between [1000:6000] pe
- cluster time within beam spill [250:1800] ns (beam-correlated)
- cluster hits > 100

The following statistics are taken as total PMT hits / total clusters identified (so occupancy around 1 indicates the PMT almost always sees 1 hit per cluster, low occupancy ~0 indicates low activity in the PMTs). I included some reference PMTs.

PMT ID | R4080 (Dec 2022) | R4449 (Jul 2023) | R4774 (Mar 2024)
-----------------------------------------------------------
332 | 0.993 | 1.009 | 1.013 (_reference PMT_)
339 | 1.215 | 1.248 | 1.311 (_reference PMT_)
404 | 0.970 | 0.961 | 0.975 (_reference PMT_)
**359  | 0.234 | 0.234 | 0.244**
**408  | 0.943 | 0.000 | 0.000**
**343  | 0.003 | 0.005 | 0.006**

So it looks like 359 is very inefficient for AmBe events but seems to record _some_ hits for the muon-like events. I think we should wait on including this PMT from the dead mask until more is known.

408 was active back in late 2022 but by mid 2023 it is dead. It should be added to the list but it should be noted that it may need to be enabled when looking at older data. 

343 has extremely low activity since 2022; it should be included in the dead list

S81D committed:
Update DeadPMTIDs_p2v7.txt

359 shows some activity for higher-energy events. We can remove it from the dead mask but more investigation is needed
